src/GraphQL.Relay/GraphQL.Relay.csproj

```
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Authors>Jason Quense</Authors>
     <VersionPrefix>0.6.0-pre3</VersionPrefix>
-    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <TargetFrameworks>net5.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```
Can we target .NET Standard 2.0 + .NET Standard 2.1 (for NRT support) instead of .NET 5.0 to match the main GraphQL.NET project?  Tests and samples can run on .NET 5.0.